conandata.yml

```diff
@@ -0,0 +1,13 @@
+sources:
+  "64.2":
+    url: "https://github.com/unicode-org/icu/releases/download/release-64-2/icu4c-64_2-src.tgz"
+    sha256: "627d5d8478e6d96fc8c90fed4851239079a561a6a8b9e48b0892f24e82d31d6c"
+  "65.1":
+    url: "https://github.com/unicode-org/icu/releases/download/release-65-1/icu4c-65_1-src.tgz"
+    sha256: "53e37466b3d6d6d01ead029e3567d873a43a5d1c668ed2278e253b683136d948"
+  "66.1":
+    url: "https://github.com/unicode-org/icu/releases/download/release-66-1/icu4c-66_1-src.tgz"
+    sha256: "52a3f2209ab95559c1cf0a14f24338001f389615bf00e2585ef3dbc43ecf0a2e"
+  "67.RC":
```

Comment:
This version `67.RC` comes through Conan, and becomes the value of the CMake variable `icu_VERSION` and the APDFL 18 CMake files use it to make the version number of the dynamic library, resulting in

```
CMake Error at pdfl/PDF/Build/Toolkit/dl-cmake/cmake_install.cmake:564 (file):
  file INSTALL cannot find
  "/Users/kam/.conan/data/icu/67.RC/kam/testing/package/4a587f25a980cb54b5c9dd36b67bd31377d25bbe/lib/libicuuc.67.RC.dylib":
  No such file or directory.
```

The given directory has a `libicuuc.67.1.dylib` in it. I think the Conan version and the version in the shared library file names have to agree.

Reply:
I'm trying out 67.0 instead - should work.

Reply:
Doesn't work - produces libraries with version 67.1 instead.

Reply:
I would guess that the `67.1` part is in the sources somewhere. Maybe it could be patched.

Reply:
IMO this is a good candidate for a "squash-and-merge"